Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs

```diff
@@ -787,7 +842,13 @@ public static void customerProfileType(customerProfileType argument)
                 if(0 <= argument.profileType) { argument.profileTypeSpecified=true;}
             }
         }
-		public static void securePaymentContainerErrorType(securePaymentContainerErrorType argument) 
+        public static void ContactDetailType(ContactDetailType argument)
+        {
+            if (null != argument)
+            {
+            }
```

Comment:
This is redundant. Add some code here or remove the if block.

Reply:
Removed the IF block.